docs/compute/filesystems.md

```diff
@@ -202,13 +202,13 @@ In the above listing:
 
 The file permissions flags are arranged in four groups, the first character of the string, followed by 3 groups of 3 characters each:
 
-·     the first character can be either  a ’-’ or a ‘d’. the ‘-’ indicates the entry is a file. The  ‘d’ indicates the entry is a directory (which can contain other files and/or directories). The ‘l’ indicate the file entry is a symbolic link to another file.
+·     the first character can be either  a ’-’ or a ‘d’. the ‘-’ indicates the entry is a file. The  ‘d’ indicates the entry is a directory (which can contain other files and/or directories). The ‘l’ indicates the file entry is a symbolic link to another file.
 
 ·      In each of the following three groups of three characters, ‘r’ indicates the file/directory is readable, ‘w’ indicates the file/directory is writable, and the ‘x’ indicates a file that is executable, or a directory that permits programs to execute within itself. These letters are also referred to as permission “bits” for historical reasons.
 
 ·      The three groups of permissions condition the actions of three different groups of users. The first (or left-most) group is the file/directory owner’s permissions, the next group (middle) is the permissions granted to members of the group associated with the file, and the last group (right-most) is permissions granted to all others (not owners or group members).
 
-Note: For files, the owner and group execute flags(bits) can occasionally be replaced with an ‘s’. In the owner’s permissions bits, an ‘x’ replaced with an ‘s’ indicates the file is executable but will execute with an effective user ID of the file owner. The ‘S’ replacing the ‘x’ in the group permission bits indicates the file can execute but with effective group  set to the group of the directory. Essentially this means that new files and directories created under this directory will inherit the group of this directory. Finally, the “other” execute bit if set to ‘t’ or ‘T’, indicates that files in this directory can only be moved or deleted by the owner of the file.
+Note: For files, the owner and group execute flags(bits) can occasionally be replaced with an ‘s’. In the owner’s permissions bits, an ‘x’ replaced with an ‘s’ indicates the file is executable but will execute with an effective user ID of the file owner. The ‘S’ replacing the ‘x’ in the group permission bits indicates the file can execute but with an effective group  set to the group of the directory. Essentially this means that new files and directories created under this directory will inherit the group of this directory. Finally, the “other” execute bit if set to ‘t’ or ‘T’, indicates that files in this directory can only be moved or deleted by the owner of the file.
 
 For a more comprehensive and detailed exposition of the UNIX file system permissions, see the Traditional Unix permissions section of [https://en.wikipedia.org/wiki/File-system_permissions](https://en.wikipedia.org/wiki/File-system_permissions).
 
```

Comment:
Can you turn the provided items in `How to turn on/off the execute bits` into a table? You could make one column `Description` and the other `Command`

Reply:
This looks great!!